grpc-examples/src/main/java/io/stargate/grpcexample/GrpcClientExecuteQuery.java

```diff
@@ -42,6 +42,119 @@ public static void main(String[] args) throws InterruptedException {
     grpcClientExecuteQuery.executeSingleQuery();
     grpcClientExecuteQuery.executeBatchQueries();
     grpcClientExecuteQuery.executeAsyncQueries();
+    grpcClientExecuteQuery.executeStreamingQuery();
+    grpcClientExecuteQuery.executeStreamingBatchQueries();
+  }
+
+  private void executeStreamingBatchQueries() throws InterruptedException {
+    executeStreamingBatchInsertQueries();
+    executeStreamingSelectQuery();
+  }
+
+  private void executeStreamingQuery() throws InterruptedException {
+    executeStreamingInsertQuery();
+    executeStreamingSelectQuery();
+  }
+
+  private void executeStreamingSelectQuery() throws InterruptedException {
+    CountDownLatch responseRetrieved = new CountDownLatch(1);
+    StreamObserver<QueryOuterClass.StreamingResponse> responseStreamObserver =
+        new StreamObserver<QueryOuterClass.StreamingResponse>() {
+          @Override
+          public void onNext(QueryOuterClass.StreamingResponse value) {
+            System.out.println("Select streaming response: " + value);
```

Comment:
should we use `println` in the examples?

Reply:
I think it is ok

Reply:
I think it's okay too.